.github/workflows/ci-backend.yml

```diff
@@ -114,26 +116,51 @@ jobs:
             - name: Install python dependencies
               if: steps.cache-backend-tests.outputs.cache-hit != 'true'
               run: |
+                  cd current
                   python -m pip install -r requirements-dev.txt
                   python -m pip install -r requirements.txt
 
             - name: Check formatting
               run: |
-                  black --check .
-                  isort --check-only .
+                  cd current
+                  black --exclude posthog/hogql/grammar --check .
+                  isort --skip posthog/hogql/grammar --check-only .
 
             - name: Check for errors and code style violations
               run: |
-                  flake8 .
+                  cd current
+                  flake8 --exclude posthog/hogql/grammar .
 
             - name: Check static typing
               run: |
+                  cd current
                   mypy .
 
             - name: Check if "schema.py" is up to date
               run: |
+                  cd current
                   npm run schema:build:python && git diff --exit-code
 
+            - name: Check if antlr definitions are up to date
+              run: |
+                  export ANTLR_VERSION=4.11.1
+
+                  sudo apt-get install default-jre
```

Comment:
Looks like in Ubuntu 20.04, ANTLR 4.7.2 is available via `apt`. Can you add a comment clarifying why we need to install 4.11.1 this way instead of just doing `apt install antlr4`? (Unless I'm missing something and we can't use `apt`, but that might be useful too)

Reply:
Ant via apt was the first thing I tried. v4.7 complained about the grammar, and generates a slightly different parser as a result, so I decided to lock it down.